src/services/game.service.ts

```diff
@@ -37,10 +37,11 @@ const prepareCorrectGuesResult = (playerOfTheDay: IPlayer) => {
 
 const prepareWrongGuesResult = async (playerOfTheDay: IPlayer, selectedPlayer: IPlayer) => {
     const player = selectedPlayer.getPlayerInfo()
+    const todayPlayer = playerOfTheDay.getPlayerInfo()
     const yearIsCorrect = selectedPlayer.comparePlayerYears(playerOfTheDay)
-    const nationalityIsCorrect = player.nationalityId === playerOfTheDay.nationalityId
-    const isTeamCorrect = player.team === playerOfTheDay.team
-    const isPositionCorrect = player.positionId === playerOfTheDay.positionId
+    const nationalityIsCorrect = player?.nationality?.nationalityId == todayPlayer?.nationality?.nationalityId
+    const isTeamCorrect = player?.team?.id == todayPlayer?.team?.id
```

Comment:
this might produce a bug if both selected player and today players team is not defined it will result with 
true that is actualy true statement, but is not correct for the bussines logic of the project

should be checked first if the teams is not null before to compare them...  same on the above line with nationality